python-lib/gluonts_forecasts/model_handler_registry.py

```diff
@@ -0,0 +1,50 @@
+from gluonts_forecasts.model_handler import (
+    TrivialIdentity,
+    SeasonalNaive,
+    AutoARIMA,
+    SeasonalTrend,
+    NPTS,
+    FeedForward,
+    DeepAR,
+    Transformer,
+    MQCNN,
+)
+
+
+class ModelHandlerRegistry:
```

Comment:
For the sake of simplicity, how about calling it `ModelRegistry`?

Reply:
I am not very familiar with the concept of registry, is this something standard used elsewhere at Dataiku? I am interested in learning more.